src/components/command-bar/commands/search/components/integrations-tab-contents/index.tsx

```diff
@@ -0,0 +1,47 @@
+import { useMemo } from 'react'
+import { IconPipeline16 } from '@hashicorp/flight-icons/svg-react/pipeline-16'
+import { useIntegrationsByProductSlugs } from 'hooks/integrations/use-integrations-by-product-slugs'
+import { getFilteredIntegrations } from 'views/product-integrations-landing/components/searchable-integrations-list/helpers/get-filtered-integrations'
+import { useCommandBar } from 'components/command-bar'
+import CustomHitsContainer from '../custom-hits-container'
+import NoResultsMessage from '../no-results-message'
+import TabContentsCta from '../tab-contents-cta'
+import { IntegrationsTabContentsProps } from './types'
+
+const IntegrationsTabContents = ({
+	currentProductTag,
+}: IntegrationsTabContentsProps) => {
+	const { currentInputValue } = useCommandBar()
+	const { integrations } = useIntegrationsByProductSlugs({
+		productSlugs: currentProductTag
+			? [currentProductTag.id]
+			: __config.dev_dot.product_slugs_with_integrations,
+	})
+
+	const filteredIntegrations = useMemo(() => {
+		return getFilteredIntegrations({
+			integrations: integrations ?? [],
+			filterQuery: currentInputValue,
+		})
+	}, [currentInputValue, integrations])
+
+	return (
+		<>
+			<CustomHitsContainer
+				integrationsHits={filteredIntegrations}
+				type="integrations"
+				noResultsSlot={<NoResultsMessage />}
+			/>
+			{currentProductTag ? (
```

Comment:
🚧 Potentially blocking, I'm not sure: I think we should avoid rendering this link for products that don't have integrations enabled. For example, for Consul the link ends up leading to a broken page:

https://user-images.githubusercontent.com/4624598/216212653-47c907b5-38d0-4d0d-b4ac-856cc063639a.mp4

Reply:
I think we should just not render the Integrations tab in these cases! It'll always say "No results match your search", so it feels misleading to make the user do extra work for nothing. I think we can revisit in a later PR if we find alignment on the UX.

Reply:
I agree, let's not show an integrations tab for products that don't have integrations.